packages/s3-credentials-endpoint/tests/test-index.js

```diff
@@ -38,30 +39,37 @@ const {
   s3credentials,
 } = require('..');
 
+const buildEarthdataLoginClient = () =>
+  new EarthdataLoginClient({
+    clientId: process.env.EARTHDATA_CLIENT_ID,
+    clientPassword: process.env.EARTHDATA_CLIENT_PASSWORD,
+    earthdataLoginUrl: 'https://uat.urs.earthdata.nasa.gov',
+    redirectUri: process.env.DISTRIBUTION_REDIRECT_ENDPOINT,
+  });
+
 test.before(async () => {
   accessTokenModel = new models.AccessToken('token');
   await accessTokenModel.createTable();
 
-  authorizationUrl = randomId('authURL');
   const stubbedAccessToken = fakeAccessTokenFactory();
+  await accessTokenModel.create(stubbedAccessToken);
 
   sinon.stub(
     EarthdataLoginClient.prototype,
     'getAccessToken'
   ).callsFake(() => stubbedAccessToken);
 
-  sinon.stub(
-    EarthdataLoginClient.prototype,
-    'getAuthorizationUrl'
-  ).callsFake(() => authorizationUrl);
+  context = {
```

Comment:
Nit/question: is there a reason not to use ava's test context  (https://github.com/avajs/ava/blob/master/docs/02-execution-context.md) for the context object on L62?

Reply:
Nope! There's no reason other than I'm still learning ava and best practices. Thanks for pointing this out, I will update my code accordingly.